NetSDK/Services/EngineEvaluator/Splitter.cs

```diff
@@ -57,7 +57,7 @@ public long Hash(string key, int seed)
             byte[] seedResult = hashAlgorithm.ComputeHash(keyToBytes, 0, keyToBytes.Length);
             var result = BitConverter.ToInt32(seedResult, 0);
 
-            return result;
+            return result & 0xFFFFFFFFL;
```

Comment:
Example: 
If result was "-1805987245" its binary representation is "1111111111111111111111111111111110010100010110101101001001010011"
After is make the operation "result & 0xFFFFFFFFL", the returned value is "2488980051"
and its binary representation is just the last 32 bytes (right):
                                                                "10010100010110101101001001010011"